_data/organizations.yaml

```diff
@@ -0,0 +1,90 @@
+cloudflare:
+  name: Cloudflare Austin
+  url: https://www.cloudflare.com/
+  careerUrl: https://www.cloudflare.com/careers/
+  location: 106 E 6th St, Austin, TX 78701, 4th Floor
+  note:
+dataworld:
+  name: data.world
+  url: https://data.world
+  careerUrl: https://data-world.breezy.hr/
+  location: N, 7000 Mopac Service Rd \#425, Austin, TX 78731
+  note:
+frog:
+  name: Frog Design
+  url: https://www.frogdesign.com/studio/austin
+  careerUrl: https://www.frogdesign.com/careers
+  location: 101 West 6th Street, Austin, TX 78701, 2nd Floor
+  note:
+erelevance:
+  name: eRelevance
+  url: http://www.erelevancecorp.com/
+  careerUrl: http://www.erelevancecorp.com/careers/
+  location:
```

Comment:
any reason to keep note/location if they're empty?

Reply:
I left them there but empty for 2 reasons: 
1. to provide a place to quickly add data in the future
2. to provide a consistent template for future contributors to copy/paste

That said, there's no harm in removing the empty fields.